packages/remark-math/readme.md

```diff
@@ -139,9 +139,9 @@ If you turn this off, you can still use two or more dollars for text math.
 ## Syntax
 
 This plugin applies a micromark extensions to parse the syntax.
-That basically follows how code works in markdown, except that dollars (`$`)
-are used instead of backticks (`` ` ``), and that two dollars instead of 3 is
-enough for blocks.
+The syntax basically follows how code works in markdown, except that dollars (`$`)
+are used instead of backticks (`` ` ``), and you must have a newline
+in the text math in order to have it be considered a math block. 
```

Comment:
Thanks!

This last part, though, is exactly like how code works.
So I don’t think it should be in this same list:

````markdown
```“inline” code```

```“block”
code
```
````

And the 2 dollars vs. 3 backticks is still true?

What about keeping the original line in, maybe like this:

```
and that 2 or more dollars instead of 3 or more backticks is enough for blocks.
```

And adding:

````
Like code, the difference between “inline” and “block”, is in the line endings:

```
$$inline$$

$$
block
$$
```
````

Reply:
I like that suggestion as it seems to be more succinct. I will see about updating the wording when I get a chance today.